django/db/models/constraints.py

```diff
@@ -94,7 +94,7 @@ class CheckConstraint(BaseConstraint):
     def __init__(
         self, *, check, name, violation_error_code=None, violation_error_message=None
     ):
-        self.check = check
```

Comment:
Unfortunately, this is a [documented attribute](https://docs.djangoproject.com/en/5.0/ref/models/constraints/#check) so we cannot freely change its name.

Reply:
It's a shame really given it's the only abstraction that doesn't conform to the `.check` convention but I flipped the problem around and used `_check` instead for the method meant to be used to perform the system checks.